common/djangoapps/student/views/login.py

```diff
@@ -44,6 +44,10 @@
 from eventtracking import tracker
 from openedx.core.djangoapps.external_auth.login_and_register import login as external_auth_login
 from openedx.core.djangoapps.external_auth.models import ExternalAuthMap
+from openedx.core.djangoapps.password_policy.utils import (
```

Comment:
Not sure if we can about this, but should all of this code be wrapped in some feature/setting so it isn't executed unless explicitly configured?

Reply:
The check_password_policy_compliance method checks for the 'COMPLIANCE_CHECKING_ENABLED' flag, so I'm not sure it's necessary to wrap everything else in extra logic.